frontend/src/pages/Graphing/GraphingEditor.tsx

```diff
@@ -619,16 +624,30 @@ export const GraphingEditor = () => {
 		},
 	})
 
+	// This function returns an object with unique keys based on the 'name' property.
+	function getUniqueKeys(data: QueryKeyTypes[]): QueryKeyTypes[] {
+		return Object.values(
+			data.reduce((obj, key: QueryKeyTypes) => {
+				if (!obj[key.name]) {
+					obj[key.name] = key
+				}
+				return obj
+			}, {} as { [key: QueryKeyTypes['name']]: QueryKeyTypes }),
+		) as QueryKeyTypes[]
+	}
```

Comment:
could also be rewritten using the `lodash` library using
https://www.geeksforgeeks.org/lodash-_-keys-method/ and https://www.geeksforgeeks.org/lodash-_-uniq-method/

Reply:
@Vadman97 yeah sure, it is a simple change. are there any other changes needed from my side